about.html

```diff
@@ -0,0 +1,202 @@
+      <section>
+        <div class="logo-container container-fluid">
+          <h2 class="h2 logo-h2">Skilly Logo</h2>
+          <hr class="line-1" />
+          <h2 class="h2 logo-explanation">
+            The logo of Skilly was designed by @Scorearts
+          </h2>
+          <h2 class="h2 skilly-container">Skilly</h2>
+        </div>
+      </section>
+      <section>
+        <div class="container-fluid">
+          <h2 class="h2 clients-title text-center font-weight-bold">
+            What Students Say About Us
+          </h2>
+          <hr class="line-1" />
+          <div class="row">
+            <div class="col-xs-12 col-sm-4">
+              <div class="card">
+                <div class="card-body text-center single-student">
+                  <div class="student-image">
+                    <img src="./images/Paul.png" alt="Hezbon image" />
+                  </div>
+                  <div class="student-content">
+                    <div class="student-info">
+                      <h4 class="card-title">Hezbon Paul</h4>
+                      <h4 class="card-status">Web Developer</h4>
+                    </div>
+                    <h4 class="student-text">
+                      Joining Skilly has helped me realize my dreams.
+                    </h4>
+                  </div>
+                </div>
+              </div>
+            </div>
+            <div class="col-xs-12 col-sm-4">
+              <div class="card">
+                <div class="card-body text-center single-student">
+                  <div class="student-image">
+                    <img src="./images/Hezbon.png" alt="Hezbon image" />
+                  </div>
+                  <div class="student-content">
+                    <div class="student-info">
+                      <h4 class="card-title">Peter Paul</h4>
+                      <h4 class="card-status">Backend Developer</h4>
+                    </div>
+                    <h4 class="student-text">
+                      Joining Skilly has helped me double my salary.
+                    </h4>
+                  </div>
+                </div>
+              </div>
+            </div>
+            <div class="col-xs-12 col-sm-4">
+              <div class="card">
+                <div class="card-body text-center single-student">
+                  <div class="student-image">
+                    <img src="./images/Shantel.png" alt="Hezbon image" />
+                  </div>
+                  <div class="student-content">
+                    <div class="student-info">
+                      <h4 class="card-title">Shantel Carlos</h4>
+                      <h4 class="card-status">Software Engineer</h4>
+                    </div>
+                    <h4 class="student-text">
+                      Skilly has given me a great skill on how to learn.
+                    </h4>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
```

Comment:
- [ ] [Optional] Kindly be noticed that you should add two articles here, you added three.